src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/entity/StrikeOffObjectionsEntity.java

```diff
@@ -16,6 +16,7 @@
         private CreatedBy createdBy;
         private LocalDateTime createdOn;
         private ObjectionStatus status;
+        private String httpRequestId;
```

Comment:
can you annotate httpRequestId with @org.springframework.data.annotation.Transient
i haven't tried it, but it should mean that it will be ignored when we convert this to Json to send back to client.
https://stackoverflow.com/a/42753469

Reply:
Will try that now